gsa/src/web/entities/container.js

```diff
@@ -526,6 +535,7 @@ class EntitiesContainer extends React.Component {
           sortBy,
           sortDir,
           onChanged: this.handleChanged,
+          onDeleted: this.handleDeleted,
```

Comment:
Not sure if this will break something
```suggestion
          onDelete: this.handleDelete,
```